tools/connector-template/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var (
+	connectorDirPath = flag.String("connector-dir-path", "", "Path where to create the new connector directory")
```

Comment:
Yes, since here we want a relative path, it depends from where the user is launching the binary. The full path will be different on everyone's machine